ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java

```diff
@@ -710,21 +704,21 @@ protected boolean isInsertInto(QBParseInfo qbp, String dest) {
    * e.g. VALUES(1,3..)
    */
   private boolean isValueClause(ASTNode select) {
-    if(select == null) {
+    if (select == null) {
```

Comment:
Please remove formatting changes from this already big PR

Reply:
IntelliJ was doing some strange formatting stuff. Removed all auto-formats. Hopefully we should not see more of this in subsequent commits.